website/src/views/Profile/Profiletest.js

```diff
@@ -2,11 +2,16 @@ import React from 'react';
 import renderer from 'react-test-renderer';
 import { Profile } from './Profile';
 import '../../utils/tests/test.css';
+import config from '../firebaseconfig';
+import firebase from 'firebase/app';
+
+firebase.initializeApp(config);
```

Comment:
This could work but maybe we should look into just mocking the firebase functionality so we get consistent data from our mock api.